chips/ibex/src/chip.rs

```diff
@@ -210,7 +210,7 @@ pub unsafe fn configure_trap_handler() {
         .write(mtvec::trap_addr.val(_start_trap_vectored as u32 >> 2) + mtvec::mode::Vectored)
 }
 
-#[link_section = ".riscv,trap_vectored"]
+#[link_section = ".riscv,trap_vecs"]
```

Comment:
Ugh, that's what I get for going quickly; I missed this one, which is the critical one, and once this is no longer a comma, we're back to the original problem from #1778.

This is going to take a little digging I think; and may fundamentally require a `cfg` given the first 5 solutions I found online.